Service-Standardization-README.md

```diff
@@ -0,0 +1,239 @@
+# Service Standardization
+
+This document specifies structural and code-level standardization steps that should be done across all the Elevate micro-services.
+
+## Structural Standardization
+
+### **Expected Micro-service Directory Structure**
+
+```
+./<service-name>/
+├── .circleci
+├── Dockerfile
+├── .dockerignore
+├── .git
+├── .gitignore
+├── README.md
+└── src
+    ├── app.js
+    ├── jsconfig.json
+    ├── node_modules
+    ├── package.json
+    └── package-lock.json
+```
+
+## Code-Level Standardization
+
+### **Packages/Tools Used**
+
+1. Prettier
+2. Eslint
+3. Module-Alias
+4. husky
+5. lint-staged
+6. JsConfig
+
+### **Installation & Setup**
+
+1.  Install Prettier & Eslint:
+
+    ```
+    ELEVATE/<service-name>/src$ npm install prettier eslint --save-dev
+    ```
+
+    -   Add **.prettierrc.json** file to root of service directory:
+
+        ```
+        ./notification/
+        ├── .prettierrc.json    <----
+        ├── README.md
+        └── src
+        ```
+
+        **.prettierrc.json** sample:
+
+        ```
+        {
+            "trailingComma": "es5",
+            "tabWidth": 4,
+            "semi": false,
+            "singleQuote": true,
+            "printWidth": 120,
+            "useTabs": true,
+            "overrides": [
+                {
+                    "files": ["*.yml", "*.yaml"],
+                    "options": {
+                        "tabWidth": 2
+                    }
+                }
+            ]
+        }
+        ```
+
+    -   Add **.eslintrc.json** file to src directory of service:
+
+        ```
+        ./notification/src
```

Comment:
@joffinjoy Change to documents

Reply:
@rakeshSgr  Assuming document service wouldn't be an apt representation of usual Elevate services, setting the service name to more generic <service-name>.